lib/active_type/util/active_record.rb

```diff
@@ -0,0 +1,9 @@
+module ActiveType
```

Comment:
This looks like it's monkey-patching `ActiveRecord::Base`, but it's embedded under a module for some reason. 

I would remove this module and rather check whether the casted object responds to `after_cast`